diff --git a/node_modules/fastify/lib/initial-config-validation.js b/node_modules/fastify/lib/initial-config-validation.js
index d8f3d6f..790af83 100644
--- a/node_modules/fastify/lib/initial-config-validation.js
+++ b/node_modules/fastify/lib/initial-config-validation.js
@@ -4,6 +4,15 @@ const validate = require('./config-validator')
 const deepClone = require('rfdc')({ circles: true, proto: false })
 const { FST_ERR_INIT_OPTS_INVALID } = require('./errors')
 
+const defaultInitOptions = {
+  bodyLimit: 1048576,
+  caseSensitive: true,
+  pluginTimeout: 10000,
+  requestIdHeader: 'request-id',
+  requestIdLogLabel: 'reqId',
+  jsonShorthand: true
+}
+
 function validateInitialConfig (options) {
   const opts = deepClone(options)
 
@@ -33,5 +42,5 @@ function deepFreezeObject (object) {
 }
 
 module.exports = validateInitialConfig
-module.exports.defaultInitOptions = validate.defaultInitOptions
+module.exports.defaultInitOptions = defaultInitOptions
 module.exports.utils = { deepFreezeObject }
diff --git a/node_modules/fastify/lib/server.js b/node_modules/fastify/lib/server.js
index 2b8b935..2060223 100644
--- a/node_modules/fastify/lib/server.js
+++ b/node_modules/fastify/lib/server.js
@@ -336,9 +336,16 @@ function getServerInstance (options, httpHandler) {
   const server = httpsOptions
     ? https.createServer(httpsOptions, httpHandler)
     : http.createServer(options.http, httpHandler)
-  server.keepAliveTimeout = options.keepAliveTimeout
-  server.requestTimeout = options.requestTimeout
-  server.setTimeout(options.connectionTimeout)
+  if (options.keepAliveTimeout) {
+    server.keepAliveTimeout = options.keepAliveTimeout
+    server.headersTimeout = Math.max(server.headersTimeout || 60000, options.keepAliveTimeout + 1000)
+  }
+  if (options.requestTimeout) {
+    server.requestTimeout = options.requestTimeout
+  }
+  if (options.connectionTimeout) {
+    server.setTimeout(options.connectionTimeout)
+  }
   // We treat zero as null(node default) so we do not pass zero to the server instance
   if (options.maxRequestsPerSocket > 0) {
     server.maxRequestsPerSocket = options.maxRequestsPerSocket
